l10n_br_fiscal/demo/product_demo.xml

```diff
@@ -264,6 +264,108 @@
         <field name="ncm_id" ref="l10n_br_fiscal.ncm_00000000"/>
     </record>
 
+    <!-- Finished Product of national origin NCM 17029000 -->
+    <record id="product_finished_nation_origin_ncm_17029000" model="product.product">
```

Comment:
Hoje nos dados de demonstração existem produtos foram localizados dos dados de demonstração do core, isso para permitir quando for instalado o modulo de produção entre outros, por exemplo já exista dados de demo como a BoM para quando for fazer a demonstração já tenha os dados para fazer o fluxo completo, não vejo com bons olhos criar novos produtos, devemos utilizar os existentes. No caso dos parceiros, criamos alguns na localização porque realmente nos dados de core não existiam parceiros do brasil, mas no caso dos produtos não acho justificável pelos motivos descritos acima.